app/lib/primer/css/layout.css

```diff
@@ -799,383 +799,6 @@
     }
 }
 
-.Layout {
```

Comment:
FYI: I removed all the `Layout` styles here and also in `app/lib/primer/css/layout.css.json` since they are now part of `Primer::Alpha::Layout`.